crates/ir/src/comp.rs

```diff
@@ -57,7 +58,7 @@ pub struct Component {
 
     // ============== Component signature ===============
     /// Attributes of the component
-    pub attrs: ast::Attributes,
+    pub attrs: utils::comp_attrs::Attrs,
```

Comment:
Any reason to expose the `comp_attrs` submodule?

Reply:
Technically each set of `Attrs` comes with three types: `Bool, Num`, and `Attrs` for boolean and numeric flags as well as a type alias for the struct with these two types of flags.

We could publicize the underlying types instead, but that would then look something like `CompAttrs, CompBool, CompNum` and `PortAttrs, PortBool, PortNum` (for port attributes). Matter of preference honestly, I don't really mind either way.